.env.example

```diff
@@ -6,6 +6,6 @@ ETH_HTTP_ENDPOINT=""
 ETH_NETWORK_ID=""
 DOT_WS_ENDPOINT=""
 BTC_HTTP_ENDPOINT=""
-CACHE_ENDPOINT=""
-PROCESSOR_ENDPOINT=""
-ARB_WS_ENDPOINT=""
\ No newline at end of file
+CACHE_ENDPOINT="FOR INTERNAL USE ONLY"
```

Comment:
The metrics using that endpoint (only cf_price_delta and cf_quote_unavailable) won't work (it will print an error)
The rest of the exporter will work as usual